### ISSUE TITLE
Callback error should be an Error instance, not a plain object

### ISSUE THREAD

**Issue by sindresorhus** [google/google-api-nodejs-client#345]
That is the Node convention and the expected output.

See: https://github.com/addyosmani/psi/issues/43

### Test-case

```js
var googleapis = require('googleapis');
var pagespeed = googleapis.pagespeedonline('v1').pagespeedapi.runpagespeed;

// intentional grammar mistake to trigger the error
pagespeed({url: 'http://google.con'}, function (err, response) {
  if (err) {
    throw err;
    //=> [object Object]
  }
});
```

It should throw the error with stack, but instead just throws `[object Object]`.

**Comment by Alaneor** [comment]
Thanks for the report!

**Comment by ryanseys** [comment]
Fixed in [`v1.1.0`](https://github.com/google/google-api-nodejs-client/releases/tag/v1.1.0)
